package.json

```diff
@@ -7,7 +7,11 @@
     "docs:dev": "NODE_OPTIONS=--openssl-legacy-provider vitepress dev main",
     "docs:build": "NODE_OPTIONS=--openssl-legacy-provider vitepress build main",
     "docs:preview": "NODE_OPTIONS=--openssl-legacy-provider vitepress preview main",
-    "docs:build-cf": "NODE_OPTIONS=--openssl-legacy-provider vitepress build main && cp _redirects dist/",
+    "docs:build-endo": "cd endo; yarn install; yarn docs:markdown-for-agoric-documentation-repo; mv api-docs ../main/reference/endo",
```

Comment:
I'm struggling to build-endo:

<details><summary>lots of errors</summary>

```
$ yarn docs:build-endo
yarn run v1.22.5
$ cd endo; yarn install; yarn docs:markdown-for-agoric-documentation-repo; mv api-docs ../main/reference/endo
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@2.3.2: The platform "linux" is incompatible with this module.
info "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
[5/5] Building fresh packages...
$ patch-package
patch-package 8.0.0
Applying patches...
@lerna/conventional-commits@5.6.2 ✔
@lerna/version@5.6.2 ✔
$ typedoc --plugin typedoc-plugin-markdown --tsconfig tsconfig.build.json
[info] Loaded plugin typedoc-plugin-markdown
[info] Converting project at ./packages/base64
[warning] No entry points were provided, this is likely a misconfiguration.
[info] Converting project at ./packages/bundle-source
[warning] BundleSourceSimple, defined in ./packages/bundle-source/src/types.js, is referenced by BundleSource but not included in the documentation.
[warning] BundleSourceWithFormat, defined in ./packages/bundle-source/src/types.js, is referenced by BundleSource but not included in the documentation.
[warning] BundleSourceWithOptions, defined in ./packages/bundle-source/src/types.js, is referenced by BundleSource but not included in the documentation.
[warning] BundleSourceGeneral, defined in ./packages/bundle-source/src/types.js, is referenced by BundleSource but not included in the documentation.
[info] Converting project at ./packages/captp
packages/ses/types.d.ts:257:16 - error TS2300: Duplicate identifier 'Compartment'.

257   export class Compartment {
                   ~~~~~~~~~~~

  ../node_modules/ses/types.d.ts:242:16
    242   export class Compartment {
                       ~~~~~~~~~~~
    'Compartment' was also declared here.

../node_modules/@endo/marshal/src/types.d.ts:1:44 - error TS2304: Cannot find name 'PassableCap'.

1 export type ConvertValToSlot<Slot> = (val: PassableCap) => Slot;
                                             ~~~~~~~~~~~

../node_modules/@endo/marshal/src/types.d.ts:2:116 - error TS2304: Cannot find name 'PassableCap'.

2 export type ConvertSlotToVal<Slot> = (slot: Slot, iface?: import('@endo/pass-style').InterfaceSpec | undefined) => PassableCap;
                                                                                                                     ~~~~~~~~~~~

../node_modules/@endo/marshal/src/types.d.ts:56:37 - error TS2304: Cannot find name 'Passable'.
```

and so on and so on
</details>

Reply:
Yeah this is where it's annoying, you'd have to `rm -rf node_modules` at repo root first before running `yarn docs:build-[endo|agoric-sdk]` individually

I documented this as `yarn //docs:build-cf` but open to any suggestions

Reply:
ah. I get it now.

I read `//docs:build-cf` but not carefully.

Maybe I can use `mv` rather than `rm` so I can avoid re-building xsnap...